### PR TITLE
Adding Task State to RunningSplitInfo

### PR DESCRIPTION
**Pull request by cemcayiroglu**
```
== NO RELEASE NOTE ==
```

It was observed that there was a long running split
after the query was abandoned/cancelled. Adding the
task state info to do better troubleshooting.

The new output of "/v1/maxActiveSplits" looks like:

6 splits have been continuously active for more than 600.00s seconds

"20200603_191419_00288_79sup.3.0.220-18" tid=103 **status=running**
  at java.base@10/java.net.SocketInputStream.socketRead0(Native Method)
  at java.base@10/java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
  at java.base@10/java.net.SocketInputStream.read(SocketInputStream.java:171)
  at java.base@10/java.net.SocketInputStream.read(SocketInputStream.java:141)
  at java.base@10/java.io.DataInputStream.readFully(DataInputStream.java:200)
  at java.base@10/java.io.DataInputStream.readFully(DataInputStream.java:170)

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java [hunk 1]
```diff
@@ -898,13 +898,15 @@ public long getRunAwaySplitCount()
         private final String threadId;
         private final Thread thread;
         private boolean printed;
+        private PrioritizedSplitRunner split;
```
Comment (highker) [inline presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java:901]: `final` and put it before `printed`

File: presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java [hunk 2]
```diff
@@ -898,13 +898,15 @@ public long getRunAwaySplitCount()
...
 
-        public RunningSplitInfo(long startTime, String threadId, Thread thread)
+        public RunningSplitInfo(long startTime, Thread thread, String threadId, PrioritizedSplitRunner split)
```
Comment (highker) [inline presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java:903]: Keep the same param order as the ones on stack

File: presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java [hunk 3]
```diff
@@ -897,14 +897,16 @@ public long getRunAwaySplitCount()
...
         private boolean printed;
 
-        public RunningSplitInfo(long startTime, String threadId, Thread thread)
+        public RunningSplitInfo(long startTime, String threadId, Thread thread, PrioritizedSplitRunner split)
         {
             this.startTime = startTime;
             this.threadId = threadId;
             this.thread = thread;
             this.printed = false;
+            this.split = split;
```
Comment (highker) [inline presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java:909]: move this above `this.printed = false;` to align with stack order

Comment (cemcayiroglu) [reply presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java:908]: done

**Comment by highker** [comment]
Could you rebase and push? Facebook integration test is failing.

**Pull request merged**